setup.py

```diff
@@ -22,9 +22,10 @@
     license='LICENSE',
     packages=['cloudify_netconf'],
     install_requires=[
-        'cloudify-common>=4.6',
+        'cloudify-common==5.1.0.dev1',
```

Comment:
Needs to stay `>=4.6` otherwise we will never be able to run it on a manager less than 5.1